packages/lab/src/__tests__/__e2e__/list-next/ListNext.cy.tsx

```diff
@@ -0,0 +1,376 @@
+    value: "item 1",
+  },
+  {
+    label:
+      "list item 2: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+    value: "item 2",
+  },
+  {
+    label:
+      "list item 3: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+    value: "item 3",
+  },
+];
+
+// Single select tests
+describe("A single select list", () => {
+  const SingleSelectList = (props: any) => {
+    return (
+      <ListNext {...props}>
+        {props.children ||
+          ITEMS.map((item, index) => {
+            return (
+              <ListItemNext key={index} value={item.label}>
+                {item.label}
+              </ListItemNext>
+            );
+          })}
+      </ListNext>
+    );
+  };
+
+  describe("list width", () => {
+    it("should render list with given width", () => {
+      cy.mount(SingleSelectList({ style: { width: "80px" } }));
+
+      cy.findByRole("listbox").should("have.css", "width", "80px");
+    });
+
+    // TODO: enable test when tooltip is ready
+    // it("should render list with long labels, wrap text and show tooltip", () => {
+    //   cy.mount(
+    //     SingleSelectList({
+    //       children: ITEMS_LONG_LABEL.map((item, index) => {
+    //         return <ListItemNext key={index}>{item.label}</ListItemNext>;
+    //       }),
+    //       style: { width: "100px" },
+    //     })
+    //   );
+
+    //   cy.findByRole("listbox").should("have.css", "width", "100px");
+    //   cy.findByRole("option", { name: ITEMS_LONG_LABEL[1].label }).trigger(
+    //     "mouseover"
+    //   );
+    //   cy.findByRole("tooltip").should("exist");
+    // });
+  });
+
+  describe("list height", () => {
+    it("should render list with 3 list items", () => {
+      cy.mount(SingleSelectList({}));
+
+      cy.findByRole("listbox").should("have.css", "height", "108px"); // 36px * 3
```

Comment:
@ZarkoPernar looks like you were reviewing an older version of test, here's the latest one: https://github.com/jpmorganchase/salt-ds/pull/1638/files#diff-02b552591bf60a02c32e17c1e8a6bfcb46ae70a984b29e2ae132f3a41256139dR28-R35

`toFixed()` was added recently bc tests were coming back with `107.98217239px` instead of `108px`